src/jit/codegenarm64.cpp

```diff
@@ -416,13 +416,17 @@ void CodeGen::genSaveCalleeSavedRegistersHelp(regMaskTP   regsToSaveMask,
                                               int         lowestCalleeSavedOffset,
                                               int         spDelta)
 {
+    assert(spDelta <= 0);
     unsigned regsToSaveCount = genCountBits(regsToSaveMask);
     if (regsToSaveCount == 0)
     {
+        if (spDelta != 0)
+        {
+            genStackPointerAdjustment(spDelta, REG_NA, nullptr);
+        }
```

Comment:
A comment would be useful. Otherwise it's not clear why it's legal for you to pass REG_NA as a temp register.

---
In reply to: [56386602](https://github.com/dotnet/coreclr/pull/3761#discussion_r56386602) [](ancestors = 56386602)